pennylane/operation.py

```diff
@@ -534,6 +535,13 @@ def parameters(self):
         """Current parameter values."""
         return self.data.copy()
 
+    @property
+    def hyperparameters(self):
+        """Non-trainable variables that define this operation."""
```

Comment:
Deleted the comment - you'll know to overwrite this logic if needed!

Reply:
```suggestion
        """dict: Dictionary of non-trainable variables that define this operation."""
```
Only outstanding comment is to add examples here for setting the hyperparameters? But this isn't required for merging.